lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -391,21 +481,46 @@ sub get_test_results {
     return $result;
 }
 
+$json_schemas{get_test_history} = joi->object->strict->props(
+    offset => joi->integer->min(0),
+    limit => joi->integer->min(0),
+    frontend_params => joi->object->strict->props(
+        domain => $zm_validator->domain_name->required,
```

Comment:
Could you update the order of the parameters, either here or in the API documentation or both, so that they match up?

Reply:
Done